.github/workflows/ci.yml

```diff
@@ -36,8 +36,15 @@ jobs:
       - name: Build with Cabal
```

Comment:
I think switching `haskell/actions/setup@v1` to `haskell-actions/setup@v2` above is a drop-in change